packages/dev/gui/src/2D/controls/button.ts

```diff
@@ -163,6 +164,14 @@ export class Button extends Rectangle {
         return true;
     }
 
+    protected _getRectangleFill(context: ICanvasRenderingContext) {
+        if (this.isEnabled) {
+            return this._getBackgroundColor(context);
+        } else {
+            return this._disabledColor;
+        }

```

Comment:
just wondering - shouldn't that be inside the `_getBackgroundColor` function instead of here?

Reply:
I think I left it there because `_disabledColor` is supposed to be only for the Button class, even through it is in Control... https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/gui/src/2D/controls/control.ts#L1181